### PR TITLE
Added working agreement: no meetings on Fridays

### PR DESCRIPTION
**Pull request by chrispine**
I realized that we have another unofficial working agreement: no meetings on Fridays! Why not make that explicit?

### CODE REVIEW

File: handbook/engineering/campaigns/index.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ For each iteration (currently one month long), we follow this process:
 
 * To avoid siloing of knowledge and to keep teammates happy, we make sure that everyone gets a chance to work in different areas of the codebase. In particular, we don't want tasks in area X to always default to person P.
 * If a process isn't serving us, we are quick to shut it down.
+* We do not schedule meeting on Fridays.
```
Comment (chrispine) [inline handbook/engineering/campaigns/index.md:86]:
```suggestion
* We do not schedule meetings on Fridays.
```

**Review by LawnGnome — APPROVED** [review]
I like it. We should wait until @eseliger is back from vacation to formalise this, but I think it's healthy, even though I'm not exactly drowning in meetings to start with.

I especially like it being Friday. :beers:

**Review by mrnugget — APPROVED** [review]

**Review by eseliger — APPROVED** [review]
Cool, but should we add that this doesn't mean you cannot pair-program or something with a colleague? This could be interpreted as "we do lonely fridays" 😁

File: handbook/engineering/campaigns/index.md [hunk 2]
```diff
@@ -83,6 +83,7 @@ For each iteration (currently one month long), we follow this process:
 
 * To avoid siloing of knowledge and to keep teammates happy, we make sure that everyone gets a chance to work in different areas of the codebase. In particular, we don't want tasks in area X to always default to person P.
 * If a process isn't serving us, we are quick to shut it down.
+* We do not schedule meetings on Fridays.
```
Comment (chrispine) [inline handbook/engineering/campaigns/index.md:86]:
```suggestion
* We do not schedule team meetings on Fridays. (Folks are free to pair on Fridays if they want.)
```

**Comment by chrispine** [comment]
> Cool, but should we add that this doesn't mean you cannot pair-program or something with a colleague? This could be interpreted as "we do lonely fridays" 😁

Updated to clarify, thanks!

**Pull request merged**